docs/manual/treefile.md

```diff
@@ -224,6 +224,12 @@ It supports the following parameters:
    also be an array of strings.  Including the same file multiple times
    is an error.
 
+ * `arch-include`: object (`Map<String,include>`), optional: Each member of this
+   object should be the name of a base architecture (`$basearch`), and the `include` value
+   functions the same as the `include` key above - it can be either
+   a single string, or an array of strings - and it has the same semantics.
+   Entries which match `arch-include` are processed after `include`.
+
```

Comment:
could we get an example like is done below for `add-files` ? For me it really came together after I looked at the related [FCOS PR](https://github.com/coreos/fedora-coreos-config/pull/149/files)

Reply:
Sure, done.

(Although of course bigger picture part of the idea here is that the FCOS config becomes also *the* reference treefile; we don't want most people writing them from scratch anyways)